pkg/rulebindingmanager/cache/cache.go

```diff
@@ -379,6 +379,11 @@ func (c *RBCache) createRule(r *typesv1.RuntimeAlertRuleBindingRule) []ruleengin
 	return []ruleengine.RuleEvaluator{}
 }
 
+// Expose the rule creator to be able to create rules from third party.
+func (c *RBCache) GetRuleCreator() ruleengine.RuleCreator {
+	return c.ruleCreator
```

Comment:
is this working? you don't need a pointer to the internal ruleCreator?

Reply:
It's an interface so it's a ptr out of the box.